### ISSUE TITLE
DataLabels cause binding error log spam in WPF

### ISSUE THREAD

**Issue by ketura** [beto-rodriguez/Live-Charts#321]
Steps to reproduce: Turn on DataLabels in the codebehind.

With every point on every line (or column), the following will be produced:

> System.Windows.Data Error: 5 : Value produced by BindingExpression is not valid for target property.; Value='<null>' BindingExpression:Path=(0); DataItem='ColumnSeries' (Name=''); target element is 'TextBlock' (Name=''); target property is 'FontFamily' (type 'FontFamily')

This also results in a somewhat noticeable slowdown, though I did not measure it.

**Comment by beto-rodriguez** [comment]
Thanks, last release I removed many bindings to increase performance, and I tried what you said, I got a different exception related with animations.

> System.Windows.Media.Animation.AnimationException was unhandled
Message: An unhandled exception of type 'System.Windows.Media.Animation.AnimationException' occurred in PresentationCore.dll
Additional information: Cannot animate the 'Left' property on a 'System.Windows.Controls.TextBlock' using a 'System.Windows.Media.Animation.DoubleAnimation'. For details see the inner exception.
